packages/frontend2/src/app/_components/table/sorting/functions/sentiment-sorting.ts

```diff
@@ -0,0 +1,36 @@
+import { type Sentiment } from '@l2beat/shared-pure'
+
+export function sortSentiments<TData extends { sentiment: Sentiment }>(
+  rowA: TData,
+  rowB: TData,
+) {
+  const sentimentOrderValueA = getSentimentOrderValue(rowA)
+  if (sentimentOrderValueA === undefined) {
+    return -1
+  }
+  const sentimentOrderValueB = getSentimentOrderValue(rowB)
+  if (sentimentOrderValueB === undefined) {
+    return 1
+  }
+
+  return sentimentOrderValueA > sentimentOrderValueB ? 1 : -1
+}
+
+export function getSentimentOrderValue<T extends { sentiment: Sentiment }>(
```

Comment:
Can't we add sth like `const sentimentOrder = ['bad', 'warning', 'neutral', 'good']` so we can do `sentimentOrder.indexOf('bad') inside`? I mean I get why this is a function, as you can call it with any object with sentiment, but this switch and if can be replaced with a oneliner. There will never be a case where we have equal sentiments, because it doesn't make sense.

Reply:
Sure, nice change